app/controllers/organizations_controller.rb

```diff
@@ -1,17 +1,18 @@
 class OrganizationsController < ApplicationController
   before_action :authenticate_user!
+  load_and_authorize_resource
   layout 'org_admin'
 
   def show
     @org = Organization.friendly.find(params[:id])
     session[:current_campaign] ||= Campaign.current_campaign_id(params[:id])
```

Comment:
What would you think about moving the `current_campaign_id` method over into the `Organization` model? You'd call it:

```
@org = Organization.friendly.find(params[:id])
@org.current_campaign # returns the current campaign, sorted by whatever makes sense
@org.current_campaigns # if we think an organization could have multiple campaigns going on (which I think we discussed was maybe not the case)
```